src/vimscript/pattern.ts

```diff
@@ -200,12 +210,15 @@ export class Pattern {
       .map((atoms) => {
         let patternString = '';
         let caseOverride: boolean | undefined;
+        let inselection: boolean | undefined;
```

Comment:
This approach (if there's a `\%V` anywhere, the whole string must be contained in the visual selection) handles the common case, but can you put a `TODO` somewhere noting that it's technically incorrect (as only that zero-length character has to be in the visual selection)?

Reply:
I wasn't aware of that... I guess I can implement it exactly as described in `:help \%V` if you think it worth it.
Otherwise, I'll add a comment.

Reply:
Nah I don't think it's worth it, at least not now. At some point I'll probably build proper support for all the weird zero-length atoms Vim supports, but for now (all this `Pattern` code is brand new), let's avoid the complexity and satisfy what's probably the only sane use case of `\%V`.